raindrop_api.py

```diff
@@ -0,0 +1,56 @@
+"""
+Module for accessing the raindrop.io API
+"""
+from datetime import datetime
+import json
+from typing import List, Tuple
+import requests
+import logging
+
```

Comment:
So here, standard Python practice is to do:

```
import logging
log = logging.getLogger(__name__)
```

And then within the module, calls are simply made to `log.debug`, `log.warn`, etc.